server/queues/tasks/CommentCreatedNotificationsTask.ts

```diff
@@ -77,6 +83,58 @@ export default class CommentCreatedNotificationsTask extends BaseTask<CommentEve
       }
     }
 
+    // send notifications to users in mentioned groups
```

Comment:
Approving for v1 but I feel like we should split this into it's own task for resiliency, especially as groups can have many members.